### ISSUE TITLE
ShanghaiTech Dataset

### ISSUE THREAD

**Issue by hdasd** [ZhihengCV/Bayesian-Crowd-Counting#33]
Hi，could you tell me how topre-Process the ShanghaiTech Dataset?thinks!

**Comment by enric1994** [comment]
I think preprocessing is not required for ShanghaiTech since the images are smaller and more uniform. You can run the training step directly
You'll probably have to apply some changes to the proposed dataloader `crowd_sh`

**Comment by 12z34x** [comment]
> I think preprocessing is not required for ShanghaiTech since the images are smaller and more uniform. You can run the training step directly
> You'll probably have to apply some changes to the proposed dataloader `crowd_sh`

I do not agree with you. The min resolution of ShanghaiTech part A <256 and the min resolution of ShanghaiTech part B <512. So you will meet crop size error if you not use preprocess code. And I think the preprocess code is sane as QNRF. But this is a trick to use 512 min resolution "preprocess code" for 256 crop size of ShanghaiTech part A.(Because you enlarge the resolution)
